sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -17,20 +17,22 @@
 
 package org.apache.spark.sql.execution.datasources.jdbc
```

Comment:
Oh heck - this did seem like the appropriate place to put this though. Any thoughts on where it could live instead?